manifest_staging/charts/aad-pod-identity/templates/nmi-daemonset.yaml

```diff
@@ -25,6 +25,12 @@ spec:
 {{ toYaml .Values.nmi.podAnnotations | indent 8 }}
 {{- end }}
     spec:
+      {{- if .Values.imagePullSecrets }}
```

Comment:
We can change this to 

```
        {{- if .Values.imagePullSecrets }}
        imagePullSecrets:
{{ toYaml .Values.imagePullSecrets | indent 8 }}
        {{- end }}
```

once we make the change in values.yaml to include `name: <image pull secret>`